tests/cross/src/test/scala/tests/hover/HoverDocSuite.scala

```diff
@@ -100,7 +100,7 @@ class HoverDocSuite extends BaseHoverSuite {
   )
 
   check(
-    "java-method",
+    "java-method".tag(IgnoreScalaVersion(_ => isJava8)),
```

Comment:
Ah nice, I didn't realize we could also do this with Java versions.

Reply:
This is kind of hack, but not much really :sweat_smile: Didn't want to introduce a whole new class tag for just Java.